lib/palindrome_permutation.rb

```diff
@@ -1,5 +1,26 @@
-
 def palindrome_permutation?(string)
-
- 
-end # method
+  # hash = {}
+  #
+  # string.each_char do |letter|
+  #   hash[letter] = string.count(letter)
+  # end
```

Comment:
Why not count the number of times a letter occurs

```suggestion
  string.each_char do |letter|
    hash[letter] = 0 if hash[letter].nil?
    hash[letter] += 1
  end
```

Reply:
Thanks! I've refactored to use this.